### ISSUE TITLE
bug: wrong number of "peer store size" in Grafana

### ISSUE THREAD

**Issue by Ivansete-status** [waku-org/nwaku#2119]
### Problem
After version v0.20.0 was deployed, an strange increase started to appear in the "Peer Storage Size" graph shown by Grafana.

![image](https://github.com/waku-org/nwaku/assets/128452529/d3d18426-f42e-47d9-b844-3b0d544c4ccf)

We need to investigate why this is happening.

Issue initially detected by @richard-ramos

**Comment by alrevuelta** [comment]
The in peers in the peerstore is kind of expected, since the peerstore is pruned every 10 minutes.
https://github.com/waku-org/nwaku/blob/master/waku/node/peer_manager/peer_manager.nim#L601

Know the `peer-store-capacity` flag of the deployment?
If not set it defaults to x times `max-connections`, so perhaps that parameter was changed in the last deployment?

**Comment by alrevuelta** [comment]
Just realized. This is expected. In https://github.com/waku-org/nwaku/commit/7d9d8a3fb4ab28ed512d3038e83fbbf66626d376#diff-d59666de8e2a09c7a948ee318df24006aa6c07130ca7a37451c60d6d8602b767L104 we changed the default amount of peers from 1.25 the max connections to 5, since 1.25 was too low.

In other words:
* Before. With max connections 300, in the past we stored up to 375 peers in the peerstore before prunning
* Now. With max connections 300, now we store up to 1500 peers before prunning. This explains the increase of peer store size.

<img width="894" alt="image" src="https://github.com/waku-org/nwaku/assets/8811422/7451070f-1b4b-4744-bcab-9b2cce97f54d">
